### PR TITLE
Obmin 271 title

### PR DESCRIPTION
**Pull request by Viktoria-Serkiz**
Created Title

### CODE REVIEW

File: src/components/title/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { H2 } from './styles';
+import { ITitle } from './types';
+
+const Title = ({ text, style } : ITitle) => (
+  <H2 style={style}>
+    {text}
+  </H2>
+);
+
+export { Title };
```
Comment (Voloshkina) [inline src/components/title/index.tsx:10]: add an empty line to the end of the file

File: src/components/title/styles.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const H2 = styled.h2`
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 24px;
+  font-family: Caveat, cursive;
+  color: #11b5ed;
```
Comment (Voloshkina) [inline src/components/title/styles.ts:9]: add an empty line before pseudo-elements

File: src/components/title/styles.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  &:before,
+  &:after {
+    display: inline-block;
+    content: '';
+    width: 11px;
+    height: 11px;
+    border-radius: 50%;
+    background-color: #11b5ed;
+    margin-bottom: 5px;
+  }
```
Comment (Voloshkina) [inline src/components/title/styles.ts:19]: the same thing

File: src/components/title/types.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { CSSProperties } from 'react';
```
Comment (Wolshebnik) [inline src/components/title/types.ts:1]: add space

File: src/components/title/args.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  text: {
+    name: 'text',
+    type: { name: 'string' },
+    description: 'Text',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: 'Title' },
+    },
+  },
+};
```
Comment (Wolshebnik) [inline src/components/title/args.ts:11]: rename file pls
arg-types.ts

**Pull request merged**